example.js

```diff
@@ -5,10 +5,11 @@ const createWindow = require('.')
 const avgOfLast3 = createWindow(3, 0)
 avgOfLast3.get() // 0
 
-avgOfLast3.push(2)
-avgOfLast3.push(3)
-avgOfLast3.push(4)
-avgOfLast3.get() // (2 + 3 + 4) / 3 = 3
+avgOfLast3.
+    push(2).
```

Comment:
I'd like to have the `.` in the beginning of the line. This makes commenting in/out and line-wise [staging](https://git-scm.com/docs/git-add) easier.

Same with `readme.md`.

Reply:
+1